tests/setup.py

```diff
@@ -15,27 +15,24 @@
 
 import re
 import sys
-from setuptools import find_packages, setup
-from setuptools.command.test import test as TestCommand
+from setuptools import find_packages, setup, Command
```

Comment:
That is incorrect import. Please use `from setuptools import test as TestCommand`

Reply:
Please ignore my above comment. that is incorrect :(